### DISCUSSION TITLE
A11y WCAG version

### DISCUSSION THREAD

**Discussion by ElenaShevchenko** [storybookjs/storybook#29368] [Help]
### Summary

Hello, which WCAG version 2.0 or 2.1 or 2.2 Axe is running inside @storybook/addon-a11y

**Comment by valentinpalkovic** [comment]
@storybook/a11y has `axe-core` defined with the version 4.2.0. When you install it freshly or clear your lock file, it will install the latest version of axe-core. Following the documentation of `axe-core` shows that WCAG versions 2.0, 2.1, and 2.2 are supported.

You can follow the documentation [here](https://storybook.js.org/addons/@storybook/addon-a11y) to set axe options on a story or global level. It is documented [here](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter-examples) for example how to only run WCAG 2.0 level rules
